Shared/sdk/SharedUtil.Tests.hpp

```diff
@@ -804,6 +804,34 @@ void SharedUtil_Hash_Tests()
         TEST_END
     }
 
+    // RSA decryption test
+    {
+        TEST_FUNCTION
+        SString rawEncodedData = Base64decode(encryptedData);
+        SString rawPrivateKey = Base64decode(privateKey);
+        assert(!rawEncodedData.empty());
+        assert(!rawPrivateKey.empty());
+        SString decryptedData = RsaDecode(rawEncodedData, rawPrivateKey);
+        assert(!decryptedData.empty());
+        assert(exptectedData == decryptedData);
+        TEST_VARS
+        const SString exptectedData;
+        const SString encryptedData;
+        const SString privateKey;
+        TEST_DATA = {
+            {"Hello world",
+             "ei2zOjHZLpp5/xhc5GyUoFIdmpj8BZFOtLOHafunis3KVYcEoZmm/JxMMnGMiAemqWCmGhkNin76Jy84G5O9pHCFR499cEvAq92Or1cX8tiM3nqJrgbeNj6SM3QWHGhnxNDaxHXbXAvh9M2dyR12+X5bQVhex5O0R2XGv1XaaRw=",
```

Comment:
Is there way to format it? We use clang formatting and avoid long lines.

Reply:
Nobody reads that code. Thats necessary

Reply:
MTA code is formatted with clang tools. And sometimes we apply formatting for all code https://github.com/multitheftauto/mtasa-blue/commit/25d9ba3d30c618ba41c06201f697c69ac9e33f7e https://github.com/multitheftauto/mtasa-blue/commit/04c7c9e6f8aabb2674a0fe8fc9bc047b9f15673c https://github.com/multitheftauto/mtasa-blue/commit/c39e2cbf9ef355e8720d98c7de7cda93fc25d735 . So it's better to fix it now.